libcaf_core/caf/logger.hpp

```diff
@@ -265,84 +202,19 @@ class CAF_CORE_EXPORT logger : public ref_counted {
   /// registered.
   static logger* current_logger();
 
-private:
+protected:
   // -- constructors, destructors, and assignment operators --------------------
 
   logger(actor_system& sys);
 
-  // -- called by the actor_system when running with a test coordinator --------
-
-  void inline_output(bool value) noexcept {
-    cfg_.inline_output = value;
-  }
-
-  // -- initialization ---------------------------------------------------------
+  virtual void init(actor_system_config& cfg) = 0;
 
-  void init(actor_system_config& cfg);
+  virtual void inline_output(bool value) = 0;
```

Comment:
Please drop. This whole inline output setup is only used by unit tests. We'll implement a separate logger for unit tests.